core-plugins/docs/HDFSMove-action.md

```diff
@@ -33,9 +33,9 @@ Example
 This example moves a file from `/source/path` to `/dest/path`:
 
     {
-        "name": "HDFSAction",
```

Comment:
rename it in the description also?
https://github.com/caskdata/hydrator-plugins/blob/78eac8424dcc51d6f6056ab2598594e930dec18a/core-plugins/src/main/java/co/cask/hydrator/plugin/batch/action/HDFSMoveAction.java#L149-L149